lab1.py

```diff
@@ -0,0 +1,41 @@
+def bubble_sort(list):
+    n = len(list)
+    swapped = True
+    while swapped:
+        swapped = False
+        for i in range(n-1):
+            if list[i] > list[i + 1]:
+                temp = list[i]
+                list[i] = list[i+1]
+                list[i+1] = temp
+                swapped = True
+
+
+def partition(list,low,high):
+    middle = list[(low + high)//2]
+    i = low-1
+    j = high+1
+    while True:
```

Comment:
while True to nienajlepszy pomysł w ogóle.
BTW Ty tutaj z tego infinit loopa w ogóle wychodzisz?